NOTICE.txt

```diff
@@ -145,70 +150,6 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
----
```

Comment:
Are these part of the codebase. Do not remove notices if they are.

Reply:
This is auto-generated, so I believe they have been removed from the codebase.